.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+This is the template for bug reports, if you have a feature request you can
+safely ignore and delete this prefilled text.
+
+Include a description of the problem: What are you trying to do? What did you
+expect?
+
```

Comment:
If possible, please also include how you were running ccdproc and any error message that was produced